### ISSUE TITLE
Support gmi schema (http://www.isotc211.org/2005/gmi) for CKAN-Next harvesting

### ISSUE THREAD

**Issue by chris-macdermaid** [GSA/data.gov#2000]
## User Story

As a user harvesting metadata records into CKAN, I want to support the gmi schema (http://www.isotc211.org/2005/gmi) so that gmi metadata records can be harvested into CKAN.

## Acceptance Criteria

GIVEN a harvest with gmi metadata is kicked off \
WHEN a harvest happens with expected errors only \
THEN the record is harvested into CKAN successfully \

## Background
Harvest source with gmi records: https://catalog-next.sandbox.datagov.us/harvest/noaa-nmfs-arko
BTW, you won't see this in the current catalog. We're switching NOAA's CSW to harvest from WAFs and we're testing this.
This looks like it's related to issues with OWSLib version support of gmi.
From the error log:
2020-08-07 17:38:57,841 ERROR [ckanext.spatial.harvesters.base] Validation errors found using profile iso19139 for object with GUID 26fe6f1b21e5250bbbaf043e950fb5c306a7acc1
2020-08-07 17:38:57,850 DEBUG [ckanext.harvest.model] Element '{http://www.isotc211.org/2005/gmi}MI_Metadata': No matching global declaration available for the validation root., line 1

## Security Considerations ([required](https://nvd.nist.gov/800-53/Rev4/control/CM-4))
None

## Sketch
[Notes or a checklist reflecting our understanding of the selected approach]

**Comment by adborden** [comment]
Thanks @chris-macdermaid , can you clarify a bit:
- Does GMI go by any other names? I'm not familiar with GMI.
- Is the lack of support specific to our version of OWSLib, or is it just unsupported by OWSLib?
- You linked to the catalog-next (CKAN 2.8) app, is this also an issue with the existing catalog (catalog.data.gov)?
- Why wasn't this an issue when we were using the CSW harvest source?

**Comment by chris-macdermaid** [comment]
@adborden  

GMI defines the markup for Geographic Metadata for imagery and gridded data (gmi).  The namespace is http://www.isotc211.org/2005/gmi/gmi.xsd.

The 0.86 version of OWSLib doesn't support GMI.  During NOAA's testing of 2.8, I was able to upgrade to the latest version. Note, there is a test that fails with that version.  This open issue goes into the details. https://github.com/ckan/ckanext-spatial/issues/200.

The last time I looked into this, Data.gov and NOAA were using a modified version OWSLib 0.86 to allow GMI to work.
